.github/workflows/add-label-when-promoted.yaml

```diff
@@ -64,8 +64,17 @@ jobs:
             echo "Label does not match the required pattern."
             echo "backport_label=false" >> $GITHUB_OUTPUT
           fi
+      - name: check if 'scylladbbot/backport_error' label was removed
+        id: check_backport_error_label
+        run: |
+          label_name="${{ github.event.label.name }}"
+          if [[ "$label_name" == scylladbbot/backport_error ]]; then
```

Comment:
just wanted to be more explicit, if we really want to check if this label was *removed*, we should probably instead put
```yaml
run: |
  if [[ "${{ github.event_name }} == 'pull_request_target' ]] && \
     [[ "${{ github.event.action }} == 'unlabeled' ]] &&
     [[ "${{ github.event.label.name == 'scylladbbot/backport_error ]]; then
    echo "backport_error_label=true" >> $GITHUB_OUTPUT
  else
    echo "backport_error_label=false" >> $GITHUB_OUTPUT
  fi
```
also, would be better to have a more readable variable name for `backport_error_label`. i'd suggest use something like "ready_for_backport".

Reply:
send a v3 to simplify it